ansible/files/multinode.sh

```diff
@@ -336,6 +336,27 @@ function upgrade_overcloud() {
   kayobe overcloud service upgrade
 }
 
+function rabbit_migration() {
+  # Ensure RabbitMQ is upgraded to 3.13
+  if kayobe overcloud host command run -l controllers -b --command "docker exec rabbitmq rabbitmqctl --version | grep 3.11" --show-output; then
```

Comment:
It's not an issue right now, but RabbitMQ versions are split into three: e.g. 3.13.7. there is a chance they'll go all the way up to 3.13.11, which would also get caught by this grep